packages/cypress/src/support.ts

```diff
@@ -24,48 +25,50 @@ afterEach(() => {
-              ...(Cypress.env('forcedColors') && { forcedColors: Cypress.env('forcedColors') }),
-              ...(Cypress.env('pauseAnimationAtEnd') && {
-                pauseAnimationAtEnd: Cypress.env('pauseAnimationAtEnd'),
-              }),
-              ...(Cypress.env('prefersReducedMotion') && {
-                prefersReducedMotion: Cypress.env('prefersReducedMotion'),
-              }),
-              ...(Cypress.env('cropToViewport') && {
-                cropToViewport: Cypress.env('cropToViewport'),
-              }),
-              ...(Cypress.env('ignoreSelectors') && {
-                ignoreSelectors: Cypress.env('ignoreSelectors'),
-              }),
+    cy.wrap(takeSnapshot(doc)).then((automaticSnapshot: CypressSnapshot) => {
+      // @ts-expect-error will fix when Cypress has its own package
+      cy.get('@manualSnapshots').then((manualSnapshots = []) => {
+        cy.url().then((url) => {
+          // pass the snapshot to the server to write to disk
+          cy.task('prepareArchives', {
+            action: 'save-archives',
+            payload: {
+              // @ts-expect-error relativeToCommonRoot is on spec (but undocumented)
+              testTitlePath: [Cypress.spec.relativeToCommonRoot, ...Cypress.currentTest.titlePath],
+              domSnapshots: [...manualSnapshots, ...(automaticSnapshot ? [automaticSnapshot] : [])],
+              chromaticStorybookParams: {
+                ...(Cypress.env('diffThreshold') && {
+                  diffThreshold: Cypress.env('diffThreshold'),
+                }),
+                ...(Cypress.env('delay') && { delay: Cypress.env('delay') }),
+                ...(Cypress.env('diffIncludeAntiAliasing') && {
+                  diffIncludeAntiAliasing: Cypress.env('diffIncludeAntiAliasing'),
+                }),
+                ...(Cypress.env('diffThreshold') && {
+                  diffThreshold: Cypress.env('diffThreshold'),
+                }),
+                ...(Cypress.env('forcedColors') && { forcedColors: Cypress.env('forcedColors') }),
+                ...(Cypress.env('pauseAnimationAtEnd') && {
+                  pauseAnimationAtEnd: Cypress.env('pauseAnimationAtEnd'),
+                }),
+                ...(Cypress.env('prefersReducedMotion') && {
+                  prefersReducedMotion: Cypress.env('prefersReducedMotion'),
+                }),
+                ...(Cypress.env('cropToViewport') && {
+                  cropToViewport: Cypress.env('cropToViewport'),
+                }),
+                ...(Cypress.env('ignoreSelectors') && {
+                  ignoreSelectors: Cypress.env('ignoreSelectors'),
+                }),
+              },
+              pageUrl: url,
+              viewport: {
+                height: Cypress.config('viewportHeight'),
+                width: Cypress.config('viewportWidth'),
+              },
+              outputDir: Cypress.config('downloadsFolder'),
```

Comment:
This entire change is really just wrapping all of this with the `cy.wrap(takeSnapshot()` on line 28.